### PR TITLE
[ci] add integration test for spec-dec vllm

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/llm/prepare.py [hunk 1]
```diff
@@ -215,6 +215,19 @@
         "option.n_positions": 1024,
         "option.model_loading_timeout": 2400,
     },
+    "tiny-llama-vllm-speculative-rb": {
+        "option.model_id": "s3://djl-llm/llama-2-tiny/",
+        "option.speculative_draft_model": "s3://djl-llm/llama-2-tiny/",
```
Comment (tosterberg) [inline tests/integration/llm/prepare.py:220]: I think this test will require a bigger model, or minimally a change to context length estimates. Tiny llama has a max embedding size of 2048, so going to 16k on the context estimates should be a no op.

Comment (sindhuvahinis) [reply tests/integration/llm/prepare.py:220]: For bigger model, we need already pre-compiled model. TinyLlama by itself takes 5 mins for compilation. Maybe I could change it 8B with tiny llama, with pre-compiled models.

**Review by tosterberg — APPROVED** [review]

**Comment by sindhuvahinis** [comment]
Waiting for new artifacts updates.

**Comment by sindhuvahinis** [comment]
This got deferred.

**Pull request closed without merging**